_data/tutorials.yaml

```diff
@@ -671,3 +671,22 @@ sliding-windows:
     ruby: disabled
     scala: disabled
     swift: disabled
+
+connect-sink-timestamp:
+  title: "Time zone conversion and Kafka Connect JDBC sink"
```

Comment:
```suggestion
  title: "Timezone conversion and Kafka Connect JDBC sink"
```

Reply:
Did you know there are technically 3 correct ways to use the word time zone. This is technically a correct usage but not when directed towards Americans :) I think it should be changed to match the usage in the rest of the tutorial

Reply:
TIL!  Agree, let's just be consistent throughout the tutorial ;)